Added migration that sets default image to placeholder.png

jonobarel opened the pull request:
Item was created without a default value.
schema now defaults the item image to "placeholder.png".

# Description

Added a rails migration for the missing default image.
This doesn't solve existing images which need to be fixed.
When creating an item with no image data it will default to "placeholder.png", as shown here:

![image](https://user-images.githubusercontent.com/4742066/198884903-eb4b33ad-feae-4e47-a864-23d98987d40e.png)

jonobarel committed:
Added migration that sets default image to placeholder.png

Item was created without a default value.
schema now defaults the item image to "placeholder.png".

vanessa-cooper commented:
Thanks jonobarel, I'll review the changes right away!

vanessa-cooper reviewed (changes requested):
Make sure all checks pass, only then you can merge the PR.

jonobarel committed:
Revert "Added migration that sets default image to placeholder.png"

This reverts commit fe32c13f0347ebedc8673fb065ff2db514b49dc7.

jonobarel committed:
Created accessor for Image field

when accessing Item.image, application will default to offering placeholder.png if no image is set.
